src/engine/renderer/tr_image_crn.cpp

```diff
@@ -38,16 +38,46 @@ class crnd_decompression_exception : public std::exception {};
 #pragma GCC diagnostic pop
 #endif
 
-bool LoadInMemoryCRN(void* buff, size_t buffLen, byte **data, int *width, int *height,
+#define CASE_CRN_FORMAT(format) \
+  case format: return STRING(format);
+
+namespace {
+std::string CRNFormatToString(crn_format format)
+{
+  switch (format)
+  {
+    CASE_CRN_FORMAT(cCRNFmtDXT1);
+    CASE_CRN_FORMAT(cCRNFmtDXT3);
+    CASE_CRN_FORMAT(cCRNFmtDXT5);
+    CASE_CRN_FORMAT(cCRNFmtDXT5_CCxY);
+    CASE_CRN_FORMAT(cCRNFmtDXT5_xGxR);
+    CASE_CRN_FORMAT(cCRNFmtDXT5_xGBR);
+    CASE_CRN_FORMAT(cCRNFmtDXT5_AGBR);
+    CASE_CRN_FORMAT(cCRNFmtDXN_XY);
+    CASE_CRN_FORMAT(cCRNFmtDXN_YX);
+    CASE_CRN_FORMAT(cCRNFmtDXT5A);
+    CASE_CRN_FORMAT(cCRNFmtETC1);
+    CASE_CRN_FORMAT(cCRNFmtETC2);
+    CASE_CRN_FORMAT(cCRNFmtETC2A);
+    CASE_CRN_FORMAT(cCRNFmtETC1S);
+    CASE_CRN_FORMAT(cCRNFmtETC2AS);
+  default:
+    return "n/a (" + std::to_string(Util::ordinal(format)) + ")";
```

Comment:
Maybe “unknown” instead of “not applicable”?

Reply:
Well, changed to "unknown" :)